packages/fluentui/state/src/managers/carouselManager.ts

```diff
@@ -0,0 +1,41 @@
+import createManager from '../createManager';
+import { Manager, ManagerConfig } from '../types';
+
+export type CarouselActions = {
+  setIndexes: (activeIndex: number | string, prevActiveIndex: number) => void;
```

Comment:
```suggestion
  setIndexes: (activeIndex: number, prevActiveIndex: number) => void;
```

_nit:_ I would prefer to have only `number`